pkg/proxy/nftables/helpers_test.go

```diff
@@ -815,3 +815,196 @@ func Test_diffNFTablesChain(t *testing.T) {
 		t.Errorf("unexpected difference in empty-chain with trailing newline:\n%s", diff)
 	}
 }
+
+// This tests tracePackets against static data, just to make sure we match things in the
```

Comment:
There is no func called `tracePackets`. Since it's copied from iptables test, should we correct both places to `tracePacket`?

Reply:
good point, let me fix this